glue/core/data_factories/astropy_table.py

```diff
@@ -40,15 +46,15 @@ def astropy_table_read(*args, **kwargs):
     # also more generally, we should first try the ASCII readers.
     if 'format' not in kwargs:
         try:
-            t = Table.read(*args, format='ascii', **kwargs)
+            t = Table.read(*args, format='ascii', encoding=encoding, **kwargs)
             # Double-check for certain FITS files that may be read in as ASCII in Python 3.11
             if not (len(t) == 1 and [c.value[0] for c in t.itercols()][:3] == ['SIMPLE', '=', 'T']):
                 return t
         except Exception:
             pass
 
     # If the above didn't work, attempt to read with no specified format
-    return Table.read(*args, **kwargs)
+    return Table.read(*args, encoding=encoding, **kwargs)
```

Comment:
```suggestion
    return Table.read(*args, **kwargs)
```
Looks like this is failing on trying VOTable format; maybe it's enough to pass encoding explicitly only for 'ascii' as that is tried first anyway?